README.md

```diff
@@ -42,6 +43,9 @@ Please follow [Drop down sort guide](/docs/README.DropDownSort.md)
 Please follow [Text search guide](docs/README.TextBoxSearch.md)
 ![Demo text search](assets/TextBoxSearch/demo.gif)
 
+### Header Sort
+Please follow [Header sort guide](docs/README.HeaderSort.md)
+![Demo header sort](assets/HeaderSort/demo.gif)
```

Comment:
All reference guides should be combines into readme, because appstore readme can not navigate to relative links

Reply:
Done